.rubocop_todo.yml

```diff
@@ -0,0 +1,275 @@
+RSpec/ExampleLength:
+  Max: 39
+
+# Offense count: 7
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: CustomTransform, IgnoredWords, DisallowedExamples.
+# DisallowedExamples: works
+RSpec/ExampleWording:
+  Exclude:
+    - 'spec/puppet_blacksmith/forge_shared.rb'
+    - 'spec/puppet_blacksmith/forge_spec.rb'
+    - 'spec/puppet_blacksmith/git_spec.rb'
+    - 'spec/puppet_blacksmith/modulefile_spec.rb'
+
+# Offense count: 1
+RSpec/ExpectInHook:
+  Exclude:
+    - 'spec/puppet_blacksmith/git_spec.rb'
+
+# Offense count: 3
+# This cop supports safe autocorrection (--autocorrect).
+RSpec/LeadingSubject:
+  Exclude:
+    - 'spec/puppet_blacksmith/modulefile_spec.rb'
+
+# Offense count: 1
+RSpec/MultipleExpectations:
+  Max: 3
+
+# Offense count: 8
+# Configuration parameters: AllowSubject.
+RSpec/MultipleMemoizedHelpers:
+  Max: 10
+
+# Offense count: 24
+# Configuration parameters: EnforcedStyle, IgnoreSharedExamples.
+# SupportedStyles: always, named_only
+RSpec/NamedSubject:
+  Exclude:
+    - 'spec/puppet_blacksmith/forge_spec.rb'
+    - 'spec/puppet_blacksmith/modulefile_spec.rb'
+
+# Offense count: 4
+# Configuration parameters: AllowedGroups.
+RSpec/NestedGroups:
+  Max: 4
+
+# Offense count: 3
+# Configuration parameters: AllowedPatterns.
+# AllowedPatterns: ^expect_, ^assert_
+RSpec/NoExpectationExample:
+  Exclude:
+    - 'spec/puppet_blacksmith/forge_shared.rb'
+    - 'spec/puppet_blacksmith/git_spec.rb'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: not_to, to_not
+RSpec/NotToNot:
+  Exclude:
+    - 'spec/puppet_blacksmith/git_spec.rb'
+
+# Offense count: 2
+RSpec/RepeatedExampleGroupDescription:
+  Exclude:
+    - 'spec/puppet_blacksmith/git_spec.rb'
+
+# Offense count: 10
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: and_return, block
+RSpec/ReturnFromStub:
+  Exclude:
+    - 'spec/puppet_blacksmith/forge_spec.rb'
+
+# Offense count: 6
+# This cop supports safe autocorrection (--autocorrect).
+RSpec/SharedExamples:
+  Exclude:
+    - 'spec/puppet_blacksmith/git_spec.rb'
+    - 'spec/puppet_blacksmith/modulefile_spec.rb'
+
+# Offense count: 1
+RSpec/SubjectStub:
+  Exclude:
+    - 'spec/puppet_blacksmith/git_spec.rb'
+
+# Offense count: 4
+# Configuration parameters: IgnoreNameless, IgnoreSymbolicNames.
+RSpec/VerifiedDoubles:
+  Exclude:
+    - 'spec/puppet_blacksmith/git_spec.rb'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+Rake/Desc:
+  Exclude:
+    - 'Rakefile'
+
+# Offense count: 1
+Security/Open:
+  Exclude:
+    - 'spec/puppet_blacksmith/git_spec.rb'
+
+# Offense count: 3
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyleForMultiline.
+# SupportedStylesForMultiline: comma, consistent_comma, no_comma
```

Comment:
These comma errors should be fairly easy to correct?  I'm a little concerned about the precedent of creating large todo files but I don't strongly object.